package.json

```diff
@@ -79,5 +79,8 @@
       },
       "default": "./lib/index.js"
     }
+  },
+  "dependencies": {
+    "@xmtp/frames-validator": "^0.4.0"
```

Comment:
Also not sure if this should be a `dependencies` or `peerDependencies` 🤔

Our leaning is to be the second

Reply:
https://github.com/xmtp/xmtp-node-js-tools/tree/main/packages/frames-validator

Reply:
If it's a `peerDependency` then we're closer to the approach in #116 where we would want the caller to pass in some instance of the validator

Reply:
The developer experience is a little smoother if you only need to depend on `onchainkit`, but both are fine

Reply:
I think is about choice, on what API to use.

OnchainKit contains some code might need React, and other it doesn't. Some code it needs Viem, and other it doesn't.

That's why we referred that as peerDependency.

In the documentation we will create a section called XMTP, where we will write that all those function need
```
yarn add @xmtp/frames-validator
```
to be used.

And then after that folks can do
```
import { getXmtpFrameMessage }  from '@coinbae/onchainkit/xmtp'
```


What do you think?

Reply:
Our goal is to make sure that folks will need the PROTO dependencies https://bundlephobia.com/package/@xmtp/frames-validator@0.4.0 only when using XMTP.

Reply:
OK. You've sold me on a peer dependency approach. That seems fine as long as it's well documented.

Then we can add something like `import { getOpenFramesMessage } from '@coinbase/onchainkit/open-frames'` as soon as there is a 1.0 version of that spec. It can call into this code for the XMTP implementation of the spec. But no need to wait on that.

Reply:
Love that